src/script_interface/observables/CylindricalLBProfileObservable.hpp

```diff
@@ -133,14 +137,16 @@ class CylindricalLBProfileObservable
 
   Variant call_method(std::string const &method,
                       VariantMap const &parameters) override {
-    if (method == "calculate") {
-      return cylindrical_profile_observable()->operator()();
-    }
-    if (method == "shape") {
-      auto const shape = cylindrical_profile_observable()->shape();
-      return std::vector<int>{shape.begin(), shape.end()};
+    if (method == "edges") {
+      auto const core_edges = cylindrical_profile_observable()->edges();
+      std::vector<Variant> variant_edges;
+      for (auto &edge : core_edges) {
+        variant_edges.push_back(Variant{edge});
```

Comment:
```c++
boost::copy(core_edges, std::back_inserter(variant_edges));
```
?

Reply:
Alternatively you can avoid the copy by moving the elements in, e.g.
```c++
for (auto &&edge :  cylindrical_profile_observable()->edges()) {
  variant_edges.push_back(edge);
}
```
should work. Then you also don't need two names.

Reply:
this works